src/components/pages/Users/Users.test.tsx

```diff
@@ -27,9 +28,9 @@ const mockUserData = getGetIdentitiesResponseMock({
     getGetIdentitiesItemResponseMock({
       id: "user1",
       addedBy: "within",
-      email: "pfft",
+      email: "pfft@example.com",
```

Comment:
I think we could create a mock function here as well for consistency reason with how we do it in other places. Also, I've spotted that we've created and use custom `getGetIdentitiesErrorMockHandler` function in `mocks/identites.ts`. From what I remember, this was a workaround in the past, however, now we have auto-generated `getGetIdentitiesMockHandler400` from `api/identities/identities.msw.ts` that we can use instead.

Reply:
In this case `getGetIdentitiesItemResponseMock` is doing exactly the same thing as our mock functions as (it generates a single `Identity`), but generated specifically for `GetIdentitiesItemResponse` (functionally identical to our mocks but not named in a generic way).

I've removed `getGetIdentitiesErrorMockHandler` and replaced it with `getGetIdentitiesMockHandler404`.